config/localization/bundles/src/bundle/org/sakaiproject/localization/bundle/content_type/content_type_names.properties

```diff
@@ -11,7 +11,7 @@ application/macwriteii=Mac Write
 application/mathematica=Mathematica
 application/ms-excel=Excel
 application/excel=Excel
-application/vnd.ms-excel.sheet.macroEnabled.12=Excel
+
```

Comment:
Can you revert these changes? You're changing the default locale.

Reply:
deleted double key

Reply:
good